js/Game.js

```diff
@@ -7,38 +7,42 @@ class Game {
   }
 
   init() {
-    game.players = Object.assign({}, domUpdates.getPlayerNames());
+    this.players = Object.assign({}, domUpdates.getPlayerNames());
     domUpdates.clearInputs();
     domUpdates.goToGameScreen();
-    return game.players;
+    return this.players;
```

Comment:
It's a little odd to return `this.players` from an `init` method. When I call `game.init()`, I wouldn't actually expect anything to be return from it - I would just expect it to do some initial setup logic. I can see you're saying `let players = game.init()` in your `index.js` file below, but that's a little unintuitive. I'd rather you take this line out of here and in your `index.js` file, simply create two lines like so:

```js
game.init();
let gamePlayers = game.players;
```